packages/@okta/vuepress-site/docs/concepts/authentication/index.md

```diff
@@ -7,80 +7,43 @@ meta:
 
 # Authentication with Okta
 
-Authentication is a crucial part of any application development.  Whether you are developing an internal IT app for your employees - or building a portal for your partners - or exposing a set of APIs for developers building apps around your resources, Okta provides the right support for your projects.
+Authentication is a crucial part in developing any application. Whether you are developing an internal IT app for your employees, building a portal for your partners, or exposing a set of APIs for developers building apps around your resources, Okta provides the right authentication support for your projects.
 
-## Building a Custom Login Experience for Your Application
+## Building a Custom Authentication Experience for Your Application
 
-The login experience is perhaps the single most important user experience any app developer needs to consider. To provide a seamless, attractive, yet secure authentication experience is not a trivial task. And typically, the login logic goes hand in hand with other features such as password reset and registration. More importantly, enhanced security in the form of strong and adaptive authentication during login is often critical to many implementations.
+The sign-in experience is one of the most important user experiences any app developer needs to consider. To provide a seamless and attractive, yet secure authentication experience is not a trivial task. Typically, the sign-in logic accompanies other features such as password reset and registration. More importantly, enhanced security in the form of strong and adaptive authentication during the sign-in process is often critical to many implementations.
```

Comment:
Should it be "...attractive, yet secure, authentication experience..."?

Reply:
No, because in this instance, the "authentication experience" is the object which is being modified by the "secure" adjective.